### ISSUE TITLE
Could u create the discussion

### ISSUE THREAD

**Issue by GeorgeS2019** [ChengYen-Tang/RL.Algorithm#2]

**Comment by GeorgeS2019** [comment]
TorchLayers
- [x] BaseFeaturesExtractor
- [x] FlattenExtractor: BaseFeaturesExtractor
- [x] NatureCNN: BaseFeaturesExtractor
- [x] MlpExtractor: nn.Module
- [x] TorchLayers.GetActorCriticArch  //get_actor_critic_arch
- [x]  TorchLayers.CreateMlp//create_mlp
- [ ] CombinedExtractor: BaseFeaturesExtractor

**Comment by GeorgeS2019** [comment]
Policies
- [x] BaseModel: nn.Module
- [x] BasePolicy: BaseModel
- [x] ActorCriticPolicy: BaseModel
- [x] ActorCriticCnnPolicy: ActorCriticPolicy
- [x]  ContinuousCritic: BaseModel
- [ ] MultiInputActorCriticPolicy: ActorCriticPolicy

**Comment by GeorgeS2019** [comment]
# Common
-[ ] Distributions
-[ ] LoggerComponents
-[ ] TorchLayers
-[ ] Preprocessing
-[ ] RunningMeanStd
-[ ] SaveUtils
-[ ] TypeAliases
-[ ] Utils

**Comment by GeorgeS2019** [comment]
## vec_env
-[ ] base_vec_env.py
-[ ] dummy_vec_env.py
-[ ] stacked_observations.py
-[ ] subproc_vec_env.py
-[ ] util.py
-[ ] vec_check_nan.py
-[ ] vec_extract_dict_obs.py
-[ ] vec_frame_stack.py
-[ ] vec_monitor.py
-[ ] vec_normalize.py
-[ ] vec_transpose.py
-[ ] vec_video_recorder.py

**Comment by GeorgeS2019** [comment]
- [ ] a2c
- [ ] common
- [ ] ddpg
- [ ] dqn
- [ ] her
- [ ] ppo
- [ ] sac
- [ ] td3

**Comment by ChengYen-Tang** [comment]
This?
![image](https://user-images.githubusercontent.com/32535494/225183849-eff4de61-529a-4eee-aba4-311bb6a13155.png)

**Comment by GeorgeS2019** [comment]
yes,   your project is ready to start getting others to contribute to speed up the progress
